kubernetes-aws-typescript/package.json

```diff
@@ -0,0 +1,12 @@
+{
+		"name": "yaml",
+		"devDependencies": {
+			"@types/node": "^14"
+		},
+		"dependencies": {
+			"typescript": "^4.0.0",
+			"@pulumi/pulumi": "^3.0.0",
+			"@pulumi/awsx": "*",
+			"@pulumi/eks": "*"
+		}
+}
```

Comment:
I know it's a codegen thing, but we should probably tidy these up to be 4 spaces. (My bad for not filing an issue on this, as I noticed it also -- I'll do that.)

Reply:
Fixed. Incorrect version reference for `@pulumi/awsx` addressed as well. Changes are in 8f2808f.